app/views/conferences/_booths.html.haml

```diff
@@ -0,0 +1,24 @@
+= content_for :splash_nav do
+  %li
+    %a.smoothscroll{ href: '#booths' } Booths
+
+.container
+  .row
+    .col-md-12.text-center
+      %h2 Booths
+  - @conference.booths.confirmed.each_slice(3).with_index do |slice, i|
+    .row.row-centered
+      - slice.each.with_index do |booth, j|
+        .col-md-4.col-sm-4.col-centered.col-top
```

Comment:
Also please work on the responsiveness, we don't want to let a mobile device render this whichever way that will make things looks ugly, and potentially confusing.